templates/confirm-email.html

```diff
@@ -0,0 +1,36 @@
+{% extends 'layouts/_base.html' %}
+{% import 'macros/helpers.html' as helpers %}
+{% from "components/button/_macro.njk" import onsButton %}
+{% from 'components/panel/_macro.njk' import onsPanel %}
+
+{% set hide_sign_out_button = content.hide_sign_out_button %}
+{% set question = content.question %}
+{% set sign_out_url = content.sign_out_url %}
+
+{% block main %}
+  {% set form = content.form %}
+  {% if form.mapped_errors %}
+    {% set error_title = _("There is a problem with this page") %}
+    {% include 'partials/error-panel.html' %}
+  {% endif %}
+
+  <h1 class="question__title u-fs-xxl" data-qa="confirm-email-title">{{ content.question.title }}</h1>
+  <legend class="fieldset__legend">{{ content.question.label}}</legend>
```

Comment:
This needs to be `<p class="checkboxes_label">{{ content.question.label}}</p>` to get same styling as in prototype.